reference/functions/body.md

```diff
@@ -0,0 +1,35 @@
+# `body()`
+
+```
+string body([string request-id])
+
+```
+
+The `body` function provides access to an upstream response body. The result is
+the raw body – if needed, parsing has to be done explictly.
+
+When called without parameter, the function returns the body for the request
+with the ID `main`.
```

Comment:
`, which is the default ID for a [request](/reference/actions/request.md)`
?

Reply:
without a parameter

oder 

without parameters